python/lsst/sims/featureScheduler/driver/config/config.py

```diff
@@ -14,106 +14,207 @@
+                                        WFD_fraction=0.44, SCP_fraction=0.08,
+                                        GP_fraction=0.004,
+                                        WFD_upper_edge_fraction=0.0,
+                                        nside=nside,
                                         generate_id_map=True)
 target_maps['r'] = fs.generate_goal_map(NES_fraction=0.46,
-                                        WFD_fraction=1.0, SCP_fraction=0.15,
-                                        GP_fraction=0.15, nside=nside,
+                                        WFD_fraction=1.0, SCP_fraction=0.08,
+                                        WFD_upper_edge_fraction=0.0,
+                                        GP_fraction=0.004,
+                                        nside=nside,
                                         generate_id_map=True)
 target_maps['i'] = fs.generate_goal_map(NES_fraction=0.46,
-                                        WFD_fraction=1.0, SCP_fraction=0.15,
-                                        GP_fraction=0.15, nside=nside,
+                                        WFD_fraction=1.0, SCP_fraction=0.08,
+                                        GP_fraction=0.004,
+                                        WFD_upper_edge_fraction=0.0,
+                                        nside=nside,
                                         generate_id_map=True)
 target_maps['z'] = fs.generate_goal_map(NES_fraction=0.4,
-                                        WFD_fraction=0.9, SCP_fraction=0.15,
-                                        GP_fraction=0.15, nside=nside,
+                                        WFD_fraction=0.9, SCP_fraction=0.08,
+                                        GP_fraction=0.004,
+                                        WFD_upper_edge_fraction=0.0,
+                                        nside=nside,
                                         generate_id_map=True)
 target_maps['y'] = fs.generate_goal_map(NES_fraction=0.,
-                                        WFD_fraction=0.9, SCP_fraction=0.15,
-                                        GP_fraction=0.15, nside=nside,
+                                        WFD_fraction=0.9, SCP_fraction=0.08,
+                                        GP_fraction=0.004,
+                                        WFD_upper_edge_fraction=0.0,
+                                        nside=nside,
                                         generate_id_map=True)
 
+cloud_map = fs.utils.generate_cloud_map(target_maps, filtername='r',
+                                        wfd_cloud_max=0.7,
+                                        scp_cloud_max=0.7,
+                                        gp_cloud_max=0.7,
+                                        nes_cloud_max=0.7)
+
+# x1 = 30.
+# x0 = 2.
+# B = x1 / (x1 - x0)
+# A = -B / x1
+# width = np.arange(x0, x1, 0.5)
+# z_pad = width + 8.
+# weight = (A * width + B)
+# height = np.zeros_like(width) + 80.
+
+width = (10.,)
```

Comment:
A comment on what these are for would be good, otherwise they are mystery magic numbers.